### PR TITLE
Add ability to customize manifest file name

### PR DESCRIPTION
**Pull request by Karakatiza666**
Useful when having multiple build configs for different browsers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +46,10 @@ Specify a fallback page for SPA mode, e.g. `index.html` or `200.html` or `404.ht
 
 If `true`, precompresses files with brotli and gzip. This will generate `.br` and `.gz` files.
 
+### manifest
+
+Specify different files for different target platforms, e.g. `chrome_manifest.json`, `firefox_manifest.json`
```
Comment (michmich112) [inline README.md]: Could you add the following information: 
"This file name must match one that is present in the `static` directory (the dir containing your static files). The selected target file will be renamed to `manifest.json` in the build directory"

File: index.js [hunk 2]
```diff
@@ -143,17 +145,17 @@ async function removeInlineScripts(directory, log) {
...
-  const sourceFilePath = join(getStaticDirectory(), "manifest.json");
+  const sourceFilePath = join(getStaticDirectory(), manifest);
   if (existsSync(sourceFilePath)) {
     log.info("Extension manifest found");
     const res = copy(sourceFilePath, join(directory, "manifest.json"));
     log.success("Successfully re-wrote extension manifest");
   } else {
     log.error(
-      "Extesion manifest not found. Make sure you've added your extension manifest in your statics directory with the name manifest.json"
+      `Extesion manifest not found. Make sure you've added your extension manifest in your statics directory with the name ${manifest}`
```
Comment (michmich112) [inline index.js]: Just noticed the typo on Extension

**Review by michmich112 — CHANGES_REQUESTED** [review]
This all looks great, just need to add a tiny bit of documentation.
Please bump `package.json` and `package-lock.json` to `1.3.0`

**Comment by michmich112** [comment]
Hey @Karakatiza666 
Love this new feature!
It will be the lead feature in v1.3.0!

I've requested some slight document additions to your PR and once fixed will be merged!

**Comment by Karakatiza666** [comment]
Thanks! Please check out the amend.

**Review by michmich112 — APPROVED** [review]

**Comment by michmich112** [comment]
Awesome! Merged and released:
Github release: https://github.com/michmich112/sveltekit-adapter-chrome-extension/releases/tag/v1.3.0
NPM package: https://www.npmjs.com/package/sveltekit-adapter-chrome-extension

Thank you for your contribution @Karakatiza666 ! Great feature enhancement

**Pull request merged**